examples/snoopread/main.go

```diff
@@ -0,0 +1,47 @@
+// Copyright - Copyleft 2019 Ripx80 All rights reserved.
```

Comment:
Please use the gopacket copyright header:

```
// Copyright 2019 The GoPacket Authors. All rights reserved.
//                                                                              
// Use of this source code is governed by a BSD-style license                   
// that can be found in the LICENSE file in the root of the source              
// tree.
```
Feel free to add your name to the AUTHORS file at the top-level as well if you'd like.

Reply:
done. thanks for joining the club!